src/main/java/com/twentyonec/ItemsLogger/commands/Commands.java

```diff
@@ -0,0 +1,67 @@
+package com.twentyonec.ItemsLogger.commands;
+
+import java.util.regex.Pattern;
+
+import org.bukkit.Bukkit;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import com.twentyonec.ItemsLogger.ItemPlayer;
+import com.twentyonec.ItemsLogger.ItemsLogger;
+import com.twentyonec.ItemsLogger.utils.ChatHandler;
+import com.twentyonec.ItemsLogger.utils.Storage;
+
+import net.md_5.bungee.api.chat.TextComponent;
+
+public class Commands implements CommandExecutor {
+
+	ItemsLogger plugin = ItemsLogger.getPlugin();
+	Storage storage = Storage.getStorage(plugin);
+
+	@Override
+	public boolean onCommand(final CommandSender sender, final Command command, final String label, final String[] args) {
+
+		if (!(sender instanceof Player))
+			return false;
+		if (args.length < 1)
+			return false;
```

Comment:
You're allowed to skip braces for single line statements, but it isn't recommended that you do.
```suggestion
		if (!(sender instanceof Player)) {
			return false;
		}
		if (args.length < 1) {
			return false;
		}
```